source/tutorials/file-sets.md

```diff
@@ -48,7 +53,7 @@ trace: /home/user (all files in directory)
 null
 ```
 
-All functions that expect a file set for an argument also accept a [path](https://nixos.org/manual/nix/stable/language/values#type-path).
+All functions that expect a file set for an argument also accept a [path](https://nixos.org/manual/nix/stable/language/values#type-path) instead.
```

Comment:
Revert, because "instead" can be misinterpreted as "mutually exclusive".

Reply:
Changing it to

> All functions that expect a file set for an argument can also accept a [path](https://nixos.org/manual/nix/stable/language/values#type-path).